addons/point_of_sale/static/src/app/store/models.js

```diff
@@ -858,8 +857,11 @@ export class Orderline extends PosModel {
         return Boolean(this.combo_parent_id || this.combo_line_ids?.length);
     }
     findAttribute(values, customAttributes) {
+        if (values.length === 0 && customAttributes.length === 0) {
+            return [];
+        }
```

Comment:
What's the point of this change?

Reply:
if there is nothing selected in values and customattributes it will loop on product attr for nothing